### ISSUE TITLE
UIN - Product Manger

### ISSUE THREAD

**Issue by jackkrooss** [tulip/ppe-logistics#191]
![image](https://user-images.githubusercontent.com/64096573/83200452-303f2680-a0f8-11ea-8a7f-50f576149787.png)

**Comment by jackkrooss** [comment]
https://docs.google.com/presentation/d/1QLCMbJh8hPtsxJZ11p6uNVNNFc_zr3KiR9e3ucT5SE8/edit#slide=id.g8746cac656_2_0

**Comment by ryuncom** [comment]
@jackkrooss 
https://three-d-corps.tulip.co/groups/QoYKyRqFGDrL9s6Wm/processes/WDyYSae6RxhcwEGTx/versions/iutLmQAowQakC7phM/steps/gM5ckpQPDhwfTJS8x
Is this the "Product Manager" this ticket is referring to?

**Comment by jackkrooss** [comment]
Yes. That is a more major remodel. Perhaps you should do an easier on like
Inventory Manager first?

**Comment by ryuncom** [comment]
Sure I will work on that one first.

**Comment by jackkrooss** [comment]
Pretty much done. Needs top line and final pass

![image](https://user-images.githubusercontent.com/64096573/84582264-c765d880-ad9e-11ea-9d3f-7060faef601c.png)
